ibis/backends/tests/tpc/ds/test_queries.py

```diff
@@ -3227,3 +3227,36 @@ def test_82(item, inventory, date_dim, store_sales):
         .order_by(_.i_item_id)
         .limit(100)
     )
+
+
+@tpc_test("ds")
+def test_97(store_sales, date_dim, catalog_sales):
+    ssci = (
+        store_sales.join(date_dim, [("ss_sold_date_sk", "d_date_sk")])
+        .filter(_.d_month_seq.between(1200, 1200 + 11))
+        .group_by(_.ss_customer_sk, _.ss_item_sk)
+        .select(_.ss_customer_sk.name("customer_sk"), _.ss_item_sk.name("item_sk"))
```

Comment:
This is incorrect. In SQL a `GROUP BY` with no aggregates is equivalent to `DISTINCT`, so you have to use the `.distinct()` method here.

I'll fix it up!